silkrpc/core/evm_executor.hpp

```diff
@@ -39,6 +39,7 @@ struct ExecutionResult {
     int64_t error_code;
     uint64_t gas_left;
     silkworm::Bytes data;
+    std::optional<std::string> preCheckErrorMessage;
```

Comment:
`preCheckErrorMessage` -> `pre_check_error{std::nullopt}`

Two little things here:
- name should be changed for naming conformance with Silkworm and the rest of Silkrpc
- `std::optional` should be initialized in conformance with Silkworm and the rest of Silkrpc

Reply:
done

Reply:
done